person.rb

```diff
@@ -1,13 +1,14 @@
 class Person < Nameable
-  attr_reader :id
-  attr_accessor :name, age
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
```

Comment:
- [OPTIONAL] It is adviseable to use shortcut for setters and getters. You can check this [article](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor`) for more info.
- [ ]  Getters and Setters for @rentals should be present

```
  attr_accessor :name, :age, :rentals

```